docs/features/decorators.rst

```diff
@@ -34,71 +34,6 @@ Please note that ''stubbed'' is a decorator generator, and cannot be used as a
 "classic" Python decorator - it must be ''\@stubbed()'', not ''\@stubbed''
 (note the parenthesis).
 
-skip_if_os
-----------
-
-``skip_if_os`` skips test based on Foreman/Satellite host os version. It
-communicates with host defined on ``robottello.properties`` to get its os
-version. Currently it checks only Red Hat Enterprise Linux versions. Example::
-
-    from robottelo.decorators.host import skip_if_os
-
-    @skip_if_os('RHEL6')
-    def test_positive_create_custom_ostree_repo(self):
-        """Create Custom ostree repository"""
-
-    @skip_if_os('RHEL6', 'RHEL5')
-    def test_negative_create_custom_ostree_repo(self):
-        """Create Custom ostree repository"""
-
-The first test will be skipped if host os is RHEL6.x.y, where x and y can be
-any number. If ``RHEL6.1`` was used as parameter it would skip for any
-RHEL6.1.z version and so on.
-
-Arbitrary number versions can be passed as parameters. On second test both RHEL
-5 and 6 families would be skipped.
-
-This decorator is used to avoid false failures when an feature is supported
-only on one os version. For example, ostree repository is available
-in RHEL7 but not in RHEL6.
-
-tier[n]
```

Comment:
I think we still use tier marks, why remove it?

Reply:
They are no longer part of `decorators` module, and don't belong in this section of the docs.

Registered marks are seen with `pytest --markers`, and our mark registration should include a help text string for documentation.